src/handlers/comment/handlers/index.ts

```diff
@@ -3,7 +3,6 @@ import { Payload, UserCommands } from "../../../types";
 import { IssueCommentCommands } from "../commands";
 import { assign } from "./assign";
 import { listAvailableCommands } from "./help";
-import { payout } from "./payout";
```

Comment:
Perhaps this should also be left as a comment.

Reply:
agree with @pavlovcik 's request